packages/cozy-harvest-lib/src/components/infos/TriggerErrorInfo.jsx

```diff
@@ -18,7 +18,9 @@ import Markdown from '../Markdown'
  */
 export class TriggerErrorInfo extends PureComponent {
   render() {
-    const { className, error, konnector, t } = this.props
+    const { className, error, forceUnknown, konnector, t } = this.props
+
+    const displayeError = forceUnknown ? new UnknownKonnectorJobError() || error
```

Comment:
Why do we need a new property `forceUnknown` ? Isn't it the responsibility of TriggerInfo to recognize that the error should be unknown ?

Reply:
s/displaye/displayed

Reply:
The case were we have a `TERMS_MISMATCH_ERROR` for a konnector with no update available is not supposed to happen. BUT as we rely on informations from two sources (error from trigger and available_version from konnector), I think we must care about the case when only the TERMS_MISMATCH_ERROR is present.

I talk with @poupotte about this point, and the expected behaviour should be to get an Unknown error displayed.

Now that you're pointing this out, maybe actually we should deal in another way, for example display a specific message for `TERMS_MISMATCH_ERROR`.